### PR TITLE
Add slackbot

### PR DESCRIPTION
**Pull request by hadim**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hadim** [comment]
@conda-forge/help-python: there is an error on the win build related to `requests`. I am not sure this related to this PR. Any idea?

Besides that, it should be ready to review and merge.

File: recipes/slackbot/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2015 Slackbot Contributors
```
Comment (beckermr) [inline recipes/slackbot/LICENSE.txt]: Please make a PR upstream to include this in the MANIFEST.in file of the upstream sdist.

Comment (hadim) [reply recipes/slackbot/LICENSE.txt]: PR is here: https://github.com/lins05/slackbot/pull/200

But the project is not very active. Could we merge this and I'll update the feedstock once the PR is merged?

Comment (beckermr) [reply recipes/slackbot/LICENSE.txt]: Yup! We should simply attempt to submit a PR. That is all.

Comment (hadim) [reply recipes/slackbot/LICENSE.txt]: Perfect. Thanks.

**Pull request merged**